packages/microcosm-www-next/src/pages/index.js

```diff
@@ -92,32 +94,37 @@ export default class IndexPage extends React.Component {
         />
 
         <section className="section">
-          <ToggleContainer
-            typeClass="-mobile"
-            microcosmView={microcosmView}
-            switchView={this.switchView}
-          />
-
+          {!bookend ? (
+            <ToggleContainer
```

Comment:
if the section is a bookend (i.e. intro or conclusion), don't show the `<ToggleContainer>`